source/templates.tex

```diff
@@ -7260,32 +7261,31 @@
 \end{codeblock}
 \end{example}
 
-When all template arguments have been deduced or obtained from
-default template arguments, all uses of template parameters in
-the template parameter list of the template and the function type
-are replaced with the corresponding deduced
-or default argument values. If the substitution results in an
-invalid type, as described above, type deduction fails.
+Both the deduced and default template argument values are substituted
+for their corresponding template parameters. If the substitution results
```

Comment:
I think this rephrasing makes it less clear that we only use default template arguments for parameters for which a template argument can't be deduced, and that we use the template arguments formed by substitution into the default template arguments rather than using the default template arguments directly. Maybe instead:

> The template arguments determined above by template argument deduction or from default template arguments are substituted for their corresponding template parameters.

or something like that?

Reply:
Thank you for taking the time to review my changes! I really appreciate it. I am sure that your response and suggestion improve my original wording, but I am away from my "home base" until tomorrow afternoon and can't give this a full analysis until then. I hope that delay is okay. Thanks again for taking the time to review. I will respond fully as soon as possible.

Will

Reply:
Upon closer review, I think that your suggestion is spot on. I will make that change when I submit a second version of this PR. Thanks again for your review!